src/tmx/TmxApi/CMakeLists.txt

```diff
@@ -63,12 +66,15 @@ ENDIF (NOT TMXAPI_NO_STATIC)
 IF (NOT TMXAPI_NO_SHARED)
     ADD_LIBRARY (${PROJECT_NAME} SHARED ${TMXAPI_SOURCES})
     IF (TMX_LIB_DIR)
-        SET_TARGET_PROPERTIES (${PROJECT_NAME} PROPERTIES LIBRARY_OUTPUT_DIRECTORY "${TMX_LIB_DIR}")
+        SET_TARGET_PROPERTIES (${PROJECT_NAME} PROPERTIES LIBRARY_OUTPUT_DIRECTORY "${TMX_LIB_DIR}" INSTALL_RPATH "/opt/carma/lib")
     ENDIF ()
     TARGET_INCLUDE_DIRECTORIES (${PROJECT_NAME} SYSTEM PUBLIC 
                                 $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+                                /opt/carma/include/carma_j2735
                                 ${Boost_INCLUDE_DIRS})
-    TARGET_LINK_LIBRARIES (${PROJECT_NAME} PUBLIC ${ASN_J2735_LIBRARIES} ${Boost_LIBRARIES} ${ICU_LIBRARIES} pugixml)
+    TARGET_LINK_LIBRARIES (${PROJECT_NAME} PUBLIC libasn_j2735 ${Boost_LIBRARIES} ${ICU_LIBRARIES} pugixml)
```

Comment:
Lets link ::carma_j2735 instead of the alias

Reply:
Addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/654/commits/ef1238953d99aa1f7aa13b59a7133c1131039ee5